components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -14,7 +14,7 @@ import { useFetchUser } from "lib/hooks/useFetchUser";
 import Checkbox from "components/atoms/Checkbox/checkbox";
 import { getTopContributorLanguages } from "lib/utils/contributor-utils";
 import { DATA_FALLBACK_VALUE } from "lib/utils/fallback-values";
-import { OscrPill } from "components/Contributors/OscrPill";
```

Comment:
I renamed the file from OscrPill to Oscr because it contains two components that share some things in that file.